### ISSUE TITLE
Don't we need the number of total tested?

### ISSUE THREAD

**Issue by yonggeun** [CSSEGISandData/COVID-19#1101]
While I was checking the tables from several countries, I realize that the trends with confirmed, deceased, and recovered could be more meaningful with the daily and accumulated number of tested from each country. You can see the example. https://www.kaggle.com/kimjihoo/coronavirusdataset#Time.csv Based on this table the number of total testees in South Korea is over 0.5% from its population. 

There is a list with the number of testees here, https://ourworldindata.org/coronavirus-testing-source-data

**Comment by alexkychen** [comment]
I agree. Including numbers of testing would be more informative than reporting only numbers of confirmed cases. For example, we could estimate how many infected cases exist in a population by the following simple equation. 

Estimated infected cases = (no. of confirmed cases / no. of testing) * population size

I am not an epidemiology expert, but this is how we ecologists would do for wild populations.

**Comment by atmacafurkan** [comment]
Some countries report official numbers, but many sites online that I can find are far behind the current dates. It would be good if number of tests are also included.
